invenio_rdm_records/fixtures/data/vocabularies.yaml

```diff
@@ -29,9 +26,6 @@ descriptiontypes:
 datetypes:
   pid-type: dat
   data-file: vocabularies/date_types.yaml
-funders:
```

Comment:
These 2 were duplicated, that's why I removed them, they are already declared in line 50.